serializers/shared/src/main/java/io/pravega/schemaregistry/serializer/shared/codec/Encoder.java

```diff
@@ -36,5 +36,5 @@
      * @param outputStream ByteArrayOutputStream where the encoded data should be written.
```

Comment:
Doc still refers to the old type. To catch these automatically during development time, you might want to use links like `{@link ByteArrayOutputStream}` in the JavaDocs.